sympy/combinatorics/fp_groups.py

```diff
@@ -516,9 +516,42 @@ def elements(self):
 
 def subgroup_quotient(G, H, fp_group=None):
     '''
-    Compute the quotient group G/H
-    where, H is the subgroup of G.
+    Compute the quotient group G/H.
+
+    Examples
+    ========
+    >>> from sympy.combinatorics.fp_groups import (FpGroup, low_index_subgroups,
+    ...                                    reidemeister_presentation, FpSubgroup)
+    >>> from sympy.combinatorics.free_groups import free_group
+    >>> from sympy.combinatorics.fp_groups import subgroup_quotient, maximal_abelian_quotient
+
+    >>> F, x, y = free_group("x, y")
+    >>> f = FpGroup(F, [x**2, y**3, (x*y)**4])
+    >>> k = f.derived_series()
+    >>> T = subgroup_quotient(f, [x, y])
+    >>> H = f.subgroup([x, y])
+    >>> T.order() == f.order()/H.order()
+    True
+
+    >>> f = FpGroup(F, [x**2, y**3, (x*y)**4])
+    >>> k = f.derived_series()
+    >>> T = subgroup_quotient(f, [x*y**2*x*y, y**2*x*y*x, y**-1])
+    >>> H = f.subgroup([x*y**2*x*y, y**2*x*y*x, y**-1])
+    >>> T.order() == f.order()/H.order()
+    True
+
     '''
+    def _get_map(fp_group, F):
```

Comment:
`_get_pres` could be a better name because you are returning a presentation rather than a map